css/main.css

```diff
@@ -84,12 +84,11 @@ body > * {
 
 /** Footer formatting **/
 .footer {
-	background-color: #000e33;
+	background-color: #343a40;
```

Comment:
I'm okay with this, but do we want to instead consider changing the navbar to match the footer ([using this](https://getbootstrap.com/docs/4.0/components/navbar/#color-schemes))? Doing this would let us keep the blue accent that we have on the footer, which I personally think might be more visually distinct than making the website more grayscale.

I'm fine with it either way though!